src/animators/baseAnimator.ts

```diff
@@ -15,7 +15,7 @@ export module Animators {
    * min(maxIterativeDelay(),
    *   max(maxTotalDuration() - duration(), 0) / <number of iterations>)
    */
-  export class Base implements PlotAnimator {
+  export class Base implements Plot {
```

Comment:
Hmm, with the rename, this class declaration looks pretty weird. Maybe use `Animators.Plot`, even though it's slightly redundant?